bot.js

```diff
@@ -172,7 +172,8 @@ bot.on("messageCreate", (msg) => {
 							// Mozilla/5.0 (compatible; <botname>/<botversion>; +<boturl>)
 						}
 					}).then((resp) => {
-						msg1.edit(resp.data.toString());
+						const messageEditedResponse = Boolean(resp.data.toString()) ? `${args[0]} is a scam` : `${args[0]} is not a scam`;
```

Comment:
What's the deal with that hot mess
This would work
```msg1.edit(resp.data?`${args[0]} is a scam`:`${args[0]} is not a scam`)```

Reply:
```js
msg1.edit(`${args[0]} is ${resp.data?"":"not"}a scam.`);```

Reply:
> ```js
> msg1.edit(`${args[0]} is ${resp.data?"":"not"}a scam.`);```
> ```

That's a pretty good option too